### DISCUSSION TITLE
API /GET user email string hidden?

### DISCUSSION THREAD

**Discussion by 1yuukuu** [kimai/kimai#4917] [General] [answered]
Hi, 
I can not GET/ the JSON response for the user email ID when I work the API/users or API/user/user_id endpoints, is the GET/ email endpoint retrieval referenced somewhere else or due to security considerations, it's hidden? thought its in POST/ PATCH/ endpoints according to the API documentation.
Thanks

**Comment by kevinpapst** [comment, marked as answer]
Yes, Email is hidden on purpose - GDPR...

**Reply by 1yuukuu** [reply]
Thank you for getting back to me so quickly, this answers my question.
